packages/common/src/lib/core/jvm/utils.ts

```diff
@@ -63,6 +63,8 @@ export function getGradleBuildFilesExtension(project: ProjectConfiguration) {
   return fileExists(getProjectFilePath(project, 'build.gradle')) ? '.gradle' : '';
 }
 
+export const getDependencyIdRegEx = () => /\s*(api|implementation|testImplementation)\s*\(?['"](?<id>[^"']+)['"]\)?/g;
```

Comment:
Could please rename the method into `getGradleDependencyIdRegEx` everywhere it appears?